instrumentation/aws-sdk/aws-sdk-2.2/library/src/main/java/io/opentelemetry/instrumentation/awssdk/v2_2/SqsMessageAccess.java

```diff
@@ -30,7 +30,7 @@
  *     href="https://github.com/aws/aws-sdk-java-v2/blob/2.2.0/services/sqs/src/main/resources/codegen-resources/service-2.json#L821-L856">Definition
  *     JSON</a>
  */
-final class SqsMessageAccess {
+public final class SqsMessageAccess {
```

Comment:
This looks like a bad idea, especially since it is not in an "internal" namespace.

Reply:
Moves the instrumentation to the AWS SDK module.